src/components/CSVForm.tsx

```diff
@@ -1,33 +1,32 @@
-import { Text } from "@gnosis.pm/safe-react-components";
-import styled from "styled-components";
+import { Box, Grid, Typography } from "@mui/material";
 
 import { CSVEditor } from "./CSVEditor";
 import { CSVUpload } from "./CSVUpload";
 import { GenerateTransfersMenu } from "./GenerateTransfersMenu";
 
-const Form = styled.div`
-  flex: 1;
-  flex-direction: column;
-  display: flex;
-  justify-content: space-around;
-  gap: 8px;
-`;
 export interface CSVFormProps {}
 
 export const CSVForm = (props: CSVFormProps): JSX.Element => {
   return (
-    <Form>
-      <Text size="xl">
-        Send arbitrarily many distinct tokens, to arbitrarily many distinct accounts with various different values from
-        a CSV file in a single transaction.
```

Comment:
Does this mean the app no longer had a short description in the App Store tile?

Reply:
No the description in the AppStore is in our manifest file